hecuba_py/tests/IStorage_tests.py

```diff
@@ -23,1042 +23,1039 @@ class IStorageTest(unittest.TestCase):
            (6917529027641081856, 8070450532247928832),
            (8070450532247928832, (2 ** 63) - 1)]
     t1024 = [-9210223345203736030,
```

Comment:
Can we generate these from a uniform distribution? Instead of having a file with tokens?